pubspec.yaml

```diff
@@ -35,6 +35,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  path_provider: ^2.1.2
```

Comment:
this is a plugin, so it requires binding initialization (afaik)

Reply:
I see, I will try to find some other solution.